docs/task_overview/introduction.md

```diff
@@ -36,50 +36,50 @@ the name of the dataset to be searched for, as defined in the analysis config is
 task to run.
 
 - {py:class}`~columnflow.tasks.calibration.CalibrateEvents`: Task to implement corrections to be
-applied on the datasets, e.g. jet-energy corrections. This task uses objects of the
-{py:class}`~columnflow.calibration.Calibrator` class to apply the calibration. The argument
-```--calibrator``` followed by the name of the Calibrator
-object to be run is needed for this task to run. A default value for this argument can be set in
-the analysis config. Similarly, the ```--shift``` argument can be given, in order to choose which
-corrections are to be used, e.g. which variation (up, down, nominal) of the jet-energy corrections
-are to be used.
+    applied on the datasets, e.g. jet-energy corrections. This task uses objects of the
```

Comment:
Could you change the breaking here so that each sentence is on one line? I think for markdown files, editors should be set up in a way similar to writing large tex files, i.e., one line per sentence and the editor wraps lines automatically at some width (but only visually).

Reply:
I'll do this for all the documentation :+1: 
One question: Should the one sentence per line be done regardless of the resulting width or should we also break after a width of 120?